src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/hyjal_trash.cpp

```diff
@@ -189,6 +195,7 @@ hyjal_trashAI::hyjal_trashAI(Creature* creature) : EscortAI(creature)
     useFlyPath = false;
     damageTaken = 0;
     memset(DummyTarget, 0, sizeof(DummyTarget));
+    SetRun();
```

Comment:
The constructor should only initialize fields, not call any ingame related code

Reply:
I create a new function to do this.Updated code.